finagle/h2/src/main/scala/com/twitter/finagle/buoyant/h2/ResponseClassifiers.scala

```diff
@@ -1,7 +1,8 @@
 package com.twitter.finagle.buoyant.h2
 
 import com.twitter.finagle.service._
-import com.twitter.util.{NonFatal, Return, Throw, Try}
+import com.twitter.util.{Return, Throw, Try}
+import scala.util.control.NonFatal
 
 object ResponseClassifiers {
```

Comment:
Yeah, they're settable via the H2 config and they feed into Finagle's retries and metrics filters.  As part of this change we want to rip all of that out and replace it with our stream aware versions.